tutorials/cloud-iot-mender-ota/image/readme.md

```diff
@@ -0,0 +1,60 @@
+# Build a Mender Yocto project OS image for Raspberry Pi3 device
+
+These steps outline how to build a Yocto Project image for a Raspberry Pi3 device. 
+
+The [Yocto Project](https://www.yoctoproject.org/) is an open source collaboration project that helps developers create custom Linux-based systems for embedded products, regardless of the hardware architecture. 
+
+The build output includes a file that can be flashed to the device storage during initial provisioning, it has suffix ."sdimg. Additionally copies of the same image with ".img” and “.bmap” suffix are uploaded to GCS bucket.
```

Comment:
to GCS bucket -> to a Cloud Storage bucket.

Replace all instances of GCS with Cloud Storage.